.github/workflows/rust.yml

```
@@ -30,124 +30,120 @@ jobs:
       if: runner.os == 'macOS'
       run: |
         brew install openssl
-    
-    - run: cargo build --workspace --verbose --locked --release
-    - run: cargo test --workspace --verbose
 
-    # save the pyrsia_node binary for the package creation
-    - name: Save pyrsia_node binary
-      uses: actions/upload-artifact@v2
+    # Install cargo deb and strip
+    - name: Install cargo-deb cargo-strip
       if: runner.os == 'Linux'
-      with:
-        name: pyrsia_node
-        path: target/release/pyrsia_node
-        if-no-files-found: error
+      uses: actions-rs/cargo@v1.0.1
+      with: 
+        command: install
+        args: cargo-deb cargo-strip --force
 
-    # save the pyrsia binary for the package creation
-    - name: Save pyrsia binary
-      uses: actions/upload-artifact@v2
-      if: runner.os == 'Linux'
-      with:
-        name: pyrsia
-        path: target/release/pyrsia
-        if-no-files-found: error
+    # Need to add build number to version number
+    - name: Iterate Build Version Number
```

Sorry for being picky... this is completely optional

```suggestion
    - name: Add Build to Version Number
```

---

> 

Many changes are needed for it all to work together.  I can break it into smaller parts but the builds will probably be broken until all the changes are in place.

---

Changed to `Add Build to Version Number`